test/fprime/fbuild/test_build.py

```diff
@@ -42,21 +42,6 @@ def get_data_dir():
     )
 
 
-def test_hash_finder():
```

Comment:
Why was this removed?  Should we not put in a test file and prove searching for hashes work?

Reply:
I initially took it out because it needed a framework path, but I added a settings.ini with a fake framework path and was able to get it working!